README.md

```diff
@@ -461,8 +321,7 @@ This section illustrates the protocol flow for the case of communication using f
 
 ```json
 {
-    "vp_token": {
-      "format": "json-ld",
+    "vp_ldp": {
       "claims":
```

Comment:
Ambiguous request - it shall either put the vp_ldp into an "id_token" or "userinfo" context.

Reply:
requesting "vp_ldp" using "userinfo". Added reqesting "auth_time" using "id_token" to illustrate  how these can work together.